ModularTegustation/tegu_items/prosthetics/scrying.dm

```diff
@@ -0,0 +1,22 @@
+/obj/item/organ/cyberimp/arm/observation
+	name = "obvservation implant"
+	desc = "A device that lets you oversee the surrounding area."
+	icon = 'ModularTegustation/tegu_items/prosthetics/icons/generic.dmi'
+	icon_state = "investigation"
+	contents = newlist(/obj/item/scrying/observation)
```

Comment:
You sure you want to give them a actual scrying orb?

Reply:
Scrying orbs for wizards can let the user ghost if they use it in hand.
They also allow the user to hear deadchat.